templates/definition/vehicle/ovms.yaml

```diff
@@ -14,6 +14,8 @@ params:
   - name: password
     required: true
   - name: vehicleid
+    description:
+      generic: Vehicle ID
     required: true
   - name: server
```

Comment:
@naltatis server sollten wir NICHT zu einem default Parameter machen, das ist hier unglücklich. Eigentlich hätte das immer `uri` heissen müssen... Daher bitte aus der defaults raus.

Reply:
Ok, aber dann muss es in das entsprechende Template wieder rein.